src/nft/components/card/media.tsx

```diff
@@ -47,7 +47,7 @@ export const StyledImage = styled.img<{
   object-fit: contain;
   visibility: ${({ $hidden }) => ($hidden ? 'hidden' : 'visible')};
   background: ${({ theme, imageLoading }) =>
-    imageLoading && `linear-gradient(270deg, ${theme.backgroundOutline} 0%, ${theme.backgroundSurface} 100%)`};
+    imageLoading && `linear-gradient(270deg, ${theme.surface3} 0%, ${theme.surface1} 100%)`};
```

Comment:
i believe this is where we're seeing the loading state issues for nft cards on dark mode

Reply:
i found that at least for the detail page it came from `interface/src/nft/css/loading.css.ts` and made the change there